diff --git a/git_externals/git_externals.py b/git_externals/git_externals.py
index 16225fc..fe06aa1 100755
--- a/git_externals/git_externals.py
+++ b/git_externals/git_externals.py
@@ -1,9 +1,19 @@
 #!/usr/bin/env python
 
 from __future__ import print_function, unicode_literals
-
-from . import __version__
-from .utils import chdir, mkdir_p, link, rm_link, git, GitError
+#
+# from . import __version__
+# from .utils import chdir, mkdir_p, link, rm_link, git, GitError
+
+if __package__ is None:
+    import sys
+    from os import path
+    sys.path.append( path.dirname( path.dirname( path.abspath(__file__) ) ) )
+    from __init__ import __version__
+    from utils import chdir, mkdir_p, link, rm_link, git, GitError
+else:
+    from . import __version__
+    from .utils import chdir, mkdir_p, link, rm_link, git, GitError
 
 import click
 import json
@@ -450,3 +460,10 @@ def enable_colored_output():
     for entry in get_entries():
         with chdir(os.path.join(DEFAULT_DIR, entry)):
             git('config', 'color.ui', 'always')
+
+
+if __name__ == '__main__':
+    from click.testing import CliRunner
+    runner = CliRunner()
+    result = runner.invoke(cli, ['update'])
+    assert result.exit_code == 0
